common/edl/asym_keys.edl

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave
+{
+    /**
+     * This enumeration defines the type of a asymmetric key.
+     * This definition is shared by the enclave and the host.
+     */
+    enum oe_asymmetric_key_type_t {
+        /**
+         * A secp256r1/NIST P-256 elliptic curve key.
+         */
+        OE_ASYMMETRIC_KEY_EC_SECP256P1 = 1,
+
+        /**
+         * Unused.
+         */
+        _OE_ASYMMETRIC_KEY_TYPE_MAX = 4294967295
```

Comment:
Is it possible to use OE_ENUM_MAX or does edger8r parser have a limitation?
If it is a limitation, it would be good to capture it in an issue and add it to backlog.

Reply:
Good catch. Thanks!

Reply:
Turns out using `OE_ENUM_MAX` did cause problem. Need to investigate it.

Reply:
Updated: should work now.